wls-gui-wahllokalsystem/src/stores/ereignisStore.ts

```diff
@@ -2,10 +2,15 @@ import type { Ereignis } from "@/types/vorfaelleundvorkommnisse/Ereignis.ts";
 import type { WahlbezirkEreignisse } from "@/types/vorfaelleundvorkommnisse/WahlbezirkEreignisse.ts";
 
 import { acceptHMRUpdate, defineStore, storeToRefs } from "pinia";
-import { ref } from "vue";
+import { computed, ref, watch, watchEffect } from "vue";
 
 import { useEreignisService } from "@/composables/vorfaelleundvorkommnisse/ereignisService.ts";
 import { useUserStore } from "@/stores/userStore.ts";
+import { useWahlbezirkStore } from "@/stores/wahlbezirkStore.ts";
+import {
+  EreignisartEnum,
+  getEreignisArtForDateRelatedToSchliessungsuhrzeit,
+} from "@/types/vorfaelleundvorkommnisse/Ereignisart.ts";
 import { WahlbezirkEreignisseBuilder } from "@/types/vorfaelleundvorkommnisse/WahlbezirkEreignisse.ts";
 
 const ereignisService = useEreignisService();
```

Comment:
```suggestion
const { getEreignisse, saveEreignisse } = useEreignisService();
```
und weiter unten dann 
~`wahlbezirkEreignisse.value = await ereignisService.getEreignisse(wahlbezirkID);`~
--> `wahlbezirkEreignisse.value = await getEreignisse(wahlbezirkID);`

und

~`await ereignisService.saveEreignisse(wahlbezirkID, wahlbezirkEreignisse.value);`~
--> `await saveEreignisse(wahlbezirkID, wahlbezirkEreignisse.value);`

Reply:
fixed mit f2aea680f3479dfa184a65096b00eda00bc8855b